### PR TITLE
Make extractOriginal() consistent with getOriginal().

### PR DESCRIPTION
**Pull request by markstory**
Make the two methods consistent. Introduce a new method for getting the original value of properties that have been modified.

I'm not very happy with the new method name, and would like suggestions if people can think of a better/clearer name.

Refs #6467

### CODE REVIEW

**Comment by antograssiot** [comment]
:+1:

**Comment by markstory** [comment]
Phpcs failures are not related but I will fix them on master.

**Comment by lorenzo** [comment]
I'm still not happy with the name either, will think of alternatives

**Comment by jhli** [comment]
Initial? Untouched?

**Comment by jhawkwind** [comment]
Would `getChangedOriginal()` and `extractChangedOriginal()` work as better names?

**Comment by lorenzo** [comment]
I think `untouched` sounds good, what do you think @markstory ?

**Comment by markstory** [comment]
@lorenzo I like that. I'll update this tonight.

**Comment by markstory** [comment]
Updated. I went with `extractOriginalChanged()`. To me `untouched` would imply the properties have not been changed, while this method gets original values for properties that _have_ changed.

File: src/Datasource/EntityTrait.php [hunk 1]
```diff
@@ -526,7 +526,10 @@ public function extract(array $properties, $onlyDirty = false)
 
     /**
      * Returns an array with the requested original properties
-     * stored in this entity, indexed by property name
+     * stored in this entity, indexed by property name.
+     *
+     * Properties that have are unchanged changed will be included in the
```
Comment (lorenzo) [inline src/Datasource/EntityTrait.php]: that have are unchanged?

Comment (markstory) [inline src/Datasource/EntityTrait.php]: I'm good at english and stuff.. :blush:

File: src/Datasource/EntityTrait.php [hunk 2]
```diff
@@ -526,7 +526,10 @@ public function extract(array $properties, $onlyDirty = false)
 
     /**
      * Returns an array with the requested original properties
-     * stored in this entity, indexed by property name
+     * stored in this entity, indexed by property name.
+     *
+     * Properties that are unchanged changed will be included in the
```
Comment (antograssiot) [inline src/Datasource/EntityTrait.php]: `unchanged changed` @markstory I guess you're tired :wine_glass:  :wink:

**Pull request merged**